services/madoc-ts/src/frontend/site/features/CanvasViewer.tsx

```diff
@@ -46,52 +54,97 @@ export const CanvasViewer: React.FC = ({ children }) => {
 
   return (
     <>
-      <OuterLayoutContainer style={{ height }}>
-        <LayoutSidebarMenu>
-          {menuItems.map(menuItem => {
-            if (menuItem.isHidden) {
-              return null;
-            }
-            return (
-              <NavIconContainer
-                key={menuItem.id}
-                $active={!menuItem.isDisabled && menuItem.id === openPanel && isOpen}
-                data-tip={menuItem.label}
-                $disabled={menuItem.isDisabled}
-                onClick={() => {
-                  if (!menuItem.isDisabled) {
-                    if (isOpen && menuItem.id === openPanel) {
-                      setIsOpen(false);
-                    } else {
-                      setIsOpen(true);
-                      setOpenPanel(menuItem.id as any);
-                      if (refs.resizableDiv.current) {
-                        refs.resizableDiv.current.scrollTop = 0;
+      <OuterLayoutContainer>
+        {tabsTop && (
+          <LayoutSidebarMenu style={{ display: 'flex' }}>
+            {menuItems.map(menuItem => {
+              if (menuItem.isHidden) {
+                return null;
+              }
+              return (
+                <>
```

Comment:
The fragment here is redundant and will produce a React key error (since they key is on the NavIconContainer)